app/uk/gov/hmrc/mobilepaye/config/GuiceModule.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mobilepaye.config
+
+import com.google.inject.name.Names.named
+import com.google.inject.{AbstractModule, TypeLiteral}
+import play.api.Mode.Mode
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.api.connector.ServiceLocatorConnector
+import uk.gov.hmrc.mobilepaye.controllers.api.ApiAccess
+import uk.gov.hmrc.mobilepaye.tasks.ServiceLocatorRegistrationTask
+import uk.gov.hmrc.play.config.ServicesConfig
+
+import scala.collection.JavaConverters._
+
+class GuiceModule(environment: Environment, configuration: Configuration) extends AbstractModule with ServicesConfig {
+
+  override protected lazy val mode: Mode = environment.mode
+  override protected lazy val runModeConfiguration: Configuration = configuration
+
+  override def configure(): Unit = {
+
+    bind(classOf[ServiceLocatorConnector]).to(classOf[ApiServiceLocatorConnector])
```

Comment:
There is a pre-canned implementation of ApiServiceLocatorConnector in the latest version of play-hmrc-api:
https://github.com/hmrc/play-hmrc-api/blob/master/src/main/scala/uk/gov/hmrc/api/connector/ServiceLocatorConnector.scala

We should probably update this everywhere under a tech debt ticket, you could change it here if you want to.